### DISCUSSION TITLE
php 8.1.5 to 8.1.6 CVE-2023-0662 published on Feb 14th

### DISCUSSION THREAD

**Discussion by fate8383** [nextcloud/all-in-one#2052] [Questions] [answered]
8.1.6 has Fixed bug GHSA-54hq-v5wp-fqgv (DOS vulnerability when parsing multipart request body). (CVE-2023-0662)
Php announced this as security advisory, will this be updated? Thanks
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-0662

**Comment by szaimen** [comment, marked as answer]
Hi, AIO v4.4.1 was just relesed which contains php updates.

**Comment by phantom2jz** [comment]
POC?
